Partition (WBP)/Functions/VI0036C.VTI0036.fun.yml

```diff
@@ -61,21 +61,41 @@
     Define Field(#LF_MYS_CODE) Reffld(#V_MYS_CODE)
     Define Field(#LF_MYS_xml) Reffld(#V_MYS_xml)
     
+    * Local Copies of Table Columns - matches following List fields
+    Define Field(#LF_SQA_ID) Reffld(#V_SQA_ID)
+    Define Field(#LF_SQA_NAME) Reffld(#V_SQA_NAME)
+    Define Field(#LF_SQA_SURNAME) Reffld(#V_SQA_SURNAME)
+    Define Field(#LF_SQA_LASTNAME) Reffld(#V_SQA_LASTNAME)
+    Define Field(#LF_SQA_DATE) Reffld(#V_SQA_DATE)
+    Define Field(#LF_SQA_SALARY) Reffld(#V_SQA_SALARY)
+    Define Field(#LF_SQA_ACTIVE) Reffld(#V_SQA_ACTIVE)
+    Define Field(#LF_SQA_DESC) Reffld(#V_SQA_DESC)
+    Define Field(#LF_SQA_TAG) Reffld(#V_SQA_TAG)
+    Define Field(#LF_SQA_TIME) Reffld(#V_SQA_TIME)
+    Define Field(#LF_SQA_RAWDATA) Reffld(#V_SQA_RAWDATA)
+    Define Field(#LF_SQA_DATA) Reffld(#V_SQA_DATA)
+    Define Field(#LF_SQA_CODE) Reffld(#V_SQA_CODE)
+    Define Field(#LF_SQA_xml) Reffld(#V_SQA_xml)
+    
     * #XG_ALL includes all Fields in the Table
     Group_By Name(#XG_ALL) Fields(#V_ID #V_NAME #V_SURNAME #V_LASTNAME #V_ACTIVE #V_CODE #V_DATE #V_DESC #V_DATA #V_RAWDATA #V_SALARY #V_TAG #V_TIME #V_XML)
     Def_List Name(#w_lOriginal) Fields(#XG_ALL) Counter(#LF_Original_Count) Type(*WORKING)
     
     Group_By Name(#XG_MysqlALL) Fields(#V_MYS_ID #V_MYS_NAME #V_MYS_SURNAME #V_MYS_LASTNAME #V_MYS_ACTIVE #V_MYS_CODE #V_MYS_DATE #V_MYS_DESC #V_MYS_DATA #V_MYS_RAWDATA #V_MYS_SALARY #V_MYS_TAG #V_MYS_TIME #V_MYS_XML)
     Def_List Name(#w_lMysqlOriginal) Fields(#XG_MysqlALL) Counter(#LF_Original_Count) Type(*WORKING)
     
+    Group_By Name(#XG_SqaALL) Fields(#V_SQA_ID #V_SQA_NAME #V_SQA_SURNAME #V_SQA_LASTNAME #V_SQA_ACTIVE #V_SQA_CODE #V_SQA_DATE #V_SQA_DESC #V_SQA_DATA #V_SQA_RAWDATA #V_SQA_SALARY #V_SQA_TAG #V_SQA_TIME #V_SQA_XML)
+    Def_List Name(#w_lSqaOriginal) Fields(#XG_SqaALL) Counter(#LF_Original_Count) Type(*WORKING)
+    
     Def_List Name(#w_lDB) Fields(#XG_ALL) Counter(#LF_DB_Count) Type(*WORKING)
     
     Use Builtin(Get_Session_Value) With_Args(DBUT) To_Get(#LF_DB_Type #LF_rc)
     
-    #Com_Owner.mthGetExecutionType()
     
     * Start testing.
     Clr_List Named(#wLRet)
+    
+    #Com_Owner.mthGetExecutionType()
```

Comment:
This needs to be after the Function Start message.

Reply:
okay